web/src/components/MediaLibrary/MediaLibrary.tsx

```diff
@@ -0,0 +1,67 @@
+import { FieldError } from '@redwoodjs/forms'
+
+import Button from 'src/components/Button/Button'
+
+declare const cloudinary: any
```

Comment:
I think using `unknown` is safer, not 100% sure tho

```suggestion
declare const cloudinary: unknown
```